lms/djangoapps/program_enrollments/rest_api/v1/tests/test_views.py

```diff
@@ -46,6 +46,7 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
 from openedx.core.djangolib.testing.utils import CacheIsolationMixin
+from social_django.models import UserSocialAuth
```

Comment:
Consider running `isort -rc lms/djangoapps/program_enrollments` -- it'll put imports like these in the right place with no work on your part.

Reply:
Bumping this

Reply:
Sorry, it got lost. Will fix.